cicd/builds_using_shipwright/overview-openshift-builds.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="overview-openshift-builds"]
+= Overview of Builds
```

Comment:
I think it would be better to be consistent with using either "Overview..." or "About...". The Pipelines and GitOps docs use _About_, so I would stick with that format.

Reply:
@michaelryanpeter, We do not want to promote or advertise Shipwright much in our doc. This is why we kept it generic (just Builds). We needed to segregate the two flavors of Builds and the segragation point was the difference between the technology. So, no need to highlight the heading in that manner.

This overview page is an equivalent page to what we have in our standalone doc. We have a category called About Builds in the standalone doc and under that category, we used this assembly `Overview of Builds`. If i rename it to About builds, then both category name and assembly name would look similar. This is why i am not in favor of renaming it to `About Builds`.

The same thing happened with Pipelines:
![image](https://github.com/openshift/openshift-docs/assets/94683525/28335e05-c1be-4de5-bb27-a16e3ff87f54)

The category name and assembly name look similar.